robottelo/rh_cloud_utils.py

```diff
@@ -83,3 +83,20 @@ def get_remote_report_checksum(org_id):
             continue
         checksum, _ = result.stdout.split(maxsplit=1)
         return checksum
+
+
+def get_report_data(report_path):
+    """Returns report data from tar file.
+
+    Args:
+        tarobj: tar file to get report data from
+    """
+    json_data = {}
+    with tarfile.open(report_path, mode='r') as tarobj:
+        for file_ in tarobj.getmembers():
+            file_name = os.path.basename(file_.name)
```

Comment:
why us os.path here instead of sticking with the Path object's methods?

Reply:
Updated the pr.